css/css-cascade/important-prop.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>CSS Cascade: Important</title>
+  <link rel="author" title="David Burns" href="http://www.theautomatedtester.co.uk">
+  <link rel="help" href="https://drafts.csswg.org/css-cascade/#importance">
+  <link rel="match" href="important-prop-ref.html">
+  <meta name="flags" content="">
+  <style>
+  @keyframes override{
+   0%,100%{background-color: #f00;}
+  }
+
+  .square {
+    color:#00f;
```

Comment:
Is it the intention that this should be !important? Otherwise I don't see the point of the test :)